docker/run_mdswshell.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Sets up a shell in a container for building and debugging minidump-stackwalk.
+
+set -e
+
+INDOCKER=${INDOCKER:-0}
+
+if [[ ${INDOCKER} -eq 1 ]]; then
```

Comment:
This script is so cool. It runs outside the docker container to set everything up, then reruns itself *inside* the docker container to finish things off. I keep meaning to rewrite the tests stuff this way, too.